tools/wptrunner/wptrunner/environment.py

```diff
@@ -46,6 +52,37 @@ def do_delayed_imports(logger, test_paths):
             (", ".join(failed), serve_root))
         sys.exit(1)
 
+def generate_hash_certificate(host: str) -> str:
+    private_key = ec.generate_private_key(ec.SECP256R1())
+    subject = issuer = x509.Name([
+        x509.NameAttribute(NameOID.COUNTRY_NAME, "DE"),
+        x509.NameAttribute(NameOID.STATE_OR_PROVINCE_NAME, "Berlin"),
+        x509.NameAttribute(NameOID.LOCALITY_NAME, "Berlin"),
+        x509.NameAttribute(NameOID.ORGANIZATION_NAME, "Wpt tests"),
+        x509.NameAttribute(NameOID.COMMON_NAME, host),
+    ])
```

Comment:
This should probably be closer to what we generate for our other certs: https://github.com/web-platform-tests/wpt/blob/b862ecfb33dcbae0804b8bbdd406388543104a69/tools/wptserve/wptserve/sslutils/openssl.py#L176-L191

Reply:
Ok, I have changed to these values. Except country code, AU is australia, I have picked AA, which is not assigned to a country according to ISO 3166-1